src/docker/base/home/lind/build/downgrade/PKGBUILD

```diff
@@ -0,0 +1,20 @@
+# shellcheck disable=SC2034
+# shellcheck disable=SC2154
+# Author: Patrick Brisbin <pbrisbin@gmail.com>
+pkgname=downgrade
+pkgver=8.0.0
+pkgrel=1
+pkgdesc="Bash script for downgrading one or more packages to a version in your cache or the A.L.A."
+arch=('any')
+url="https://github.com/pbrisbin/$pkgname"
+license=('GPL')
+source=("downgrade-v$pkgver.tar.gz::https://github.com/pbrisbin/$pkgname/archive/v$pkgver.tar.gz")
+depends=('pacman-contrib') # pacsort
+optdepends=('sudo: for installation via sudo')
+
+package() {
+  cd "$pkgname-$pkgver" || exit 1
+
+  make DESTDIR="$pkgdir" PREFIX=/usr install
+}
+md5sums=('56c1808604a51d713661cb0f3f61ec60')
```

Comment:
It's probably not a big deal, but it might be worth subrepoing the repo on the aur rather than storing a local version of the PKGBUILD in case of updates or changes of host

Reply:
done.